Tutorials/Lesson1/README.md

```diff
@@ -26,14 +26,20 @@ Thinking about the function `deposit`, a basic property is:
 The rule in [integrityOfDeposit.spec](IntegrityOfDeposit.spec) verifies this property. 
 It verifies that the `deposit` operation increases the funds of `msg.sender` by the deposited amount.  
 Formal verification can provide complete coverage of the input space, giving guarantees beyond what is possible from testing alone.
-This means that all possible inputs to the deposit function (all possible deposited amounts) are taken into account. 
-Additionally, all possible calling contexts are taken into account. Certora prover represents the calling context through the struct variable `env`. Declaring a single `env` variable suffices to capture all aspects of the calling contexts, but they can also be addressed individually.
+All possible inputs to the deposit function (all possible deposited amounts) are taken into account. 
+Additionally, all possible calling contexts are considered. Certora prover represents the calling context through the struct variable `env`. Declaring a single `env` variable suffices to capture all aspects of the calling contexts, but they can also be addressed individually.
 Some example aspects of the calling context are:
- - "who is the depositor?" (`env.msg.sender`)
- - "what was the initial balance of the depositor?" (`env.msg.sender.balance`)
- - "in which block does the deposit occur?" (`env.block.number`)
- - "at which time does the deposit occur?" (`env.block.timestamp`)
+ - "Who is the depositor?" (`env.msg.sender`)
+ - "What was the initial balance of the depositor?" (`env.msg.sender.balance`)
```

Comment:
env.msg.sender.balance  is a nice feature to add  but still not supported. Let's remove for now